openpype/hosts/nuke/plugins/publish/help/validate_asset_context.xml

```diff
@@ -3,10 +3,13 @@
     <error id="main">
         <title>Shot/Asset name</title>
         <description>
-## Invalid Shot/Asset name in subset
+## Invalid node context keys and values
 
-Following Node with name `{node_name}`:
-Is in context of `{correct_name}` but Node _asset_ knob is set as `{wrong_name}`.
+Following Node with name: \`{node_name}\`
+
+Context keys and values: \`{correct_values}\`
+
+Wrong keys and values: \`{wrong_values}\`.
 
```

Comment:
The error message feels a bit abstract to artists - no?

This is what a similar [Maya validator](https://github.com/pypeclub/OpenPype/blob/e809050f15c98e965b5561249e8c7b604a3e15c9/openpype/hosts/maya/plugins/publish/validate_instance_in_context.py#L51-L57) reports (even though that only validates asset - not task)

> ## Publishing to a different asset
> 
> There are publish instances present which are publishing into a different asset than your current context.
> 
> Usually this is not what you want but there can be cases where you might want to publish into another asset or shot. 
> 
> If that's the case you can disable the validation on the instance to ignore it.

Reply:
yes, this is much better, thanks! ecf144993fb48c0e6d9d8e7c8c0512f805abafe6